apps/lib/p5play/p5.play.js

```diff
@@ -448,6 +448,15 @@ p5.prototype._isMouseButtonInState = function(buttonCode, state) {
   if(buttonCode === undefined)
     buttonCode = this.LEFT;
 
+  // Add other acceptable strings
```

Comment:
It would be nice to make this follow the pattern used with keys, with a mapping between the string names and the constants and a lookup.